.changeset/sour-coats-shake.md

```diff
@@ -0,0 +1,5 @@
+---
+"@biomejs/biome": patch
+---
+
+CLI flag `--javascript-object-wrap` has been renamed to `--javascript-formatter-object-wrap` to keep consistency in flags.
```

Comment:
Considering the fact that `*-object-wrap` was introduced but not yet released to the users, it's not needed to add a changeset. You might want to consider updating `gentle-spies-drum.md` (the changeset where you created the feature), and update it with with the correct CLI arguments

Reply:
Updated in [7a852be](https://github.com/biomejs/biome/pull/5166/commits/7a852be9acda9b1876b1475106ef00fdc38ee5cb), the CLI flag is not introduced in the changeset so I added to the previous changeset